controllers/activemqartemis_reconciler.go

```diff
@@ -1229,6 +1232,44 @@ func (reconciler *ActiveMQArtemisReconcilerImpl) deleteRequestedResource(customR
 	return deleteError
 }
 
+func checkExistingPersistentVolumes(instance *brokerv1beta1.ActiveMQArtemis, client rtclient.Client) {
+	var log = ctrl.Log.WithName("controller_v1beta1activemqartemis")
+
+	var i int32
+	for i = 0; i < instance.Spec.DeploymentPlan.Size; i++ {
+		ordinalString := strconv.Itoa(int(i))
+		pvcKey := types.NamespacedName{Namespace: instance.Namespace, Name: instance.Name + "-" + namer.CrToSS(instance.Name) + "-" + ordinalString}
+		pvc := &corev1.PersistentVolumeClaim{}
+		err := client.Get(context.TODO(), pvcKey, pvc)
+
+		if err == nil {
+			totalLen := len(pvc.OwnerReferences)
+			if totalLen > 0 {
+				found := -1
+				newOwnerReferences := make([]metav1.OwnerReference, 0)
+				for i, oref := range pvc.OwnerReferences {
```

Comment:
This will work.

small things, but I would not reuse i, just use a bool for found, and I don't see totalLen reused, so just use len() in the if statement. Just to make it easier to understand.

Reply:
+1